PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/core/FunctionUtils.java

```diff
@@ -312,29 +313,73 @@ private static void processParameterAnnotations(final PsiMethod method, final Li
         return bindings;
     }
 
-    private static Binding getBinding(JvmAnnotation annotation) throws AzureExecutionException {
+    private static Binding getBinding(final Project project, JvmAnnotation annotation) throws AzureExecutionException {
         if (annotation == null) {
             return null;
         }
+        if (!(annotation instanceof PsiAnnotation)) {
+            PluginUtil.showWarningNotificationProject(project,
+                                                      "Cannot parse annotation information.",
+                                                      String.format("Expect type: %s, actual type: %s",
+                                                                    PsiAnnotation.class.getCanonicalName(),
+                                                                    annotation.getClass().getCanonicalName()));
+            return null;
+        }
+
         final BindingEnum annotationEnum =
                 Arrays.stream(BindingEnum.values())
                       .filter(bindingEnum -> StringUtils.equalsIgnoreCase(bindingEnum.name(),
                                                                           FilenameUtils.getExtension(annotation.getQualifiedName())))
```

Comment:
`FilenameUtils.getExtension` -> `ClassUtils.getShortClassName()` For better understanding

Reply:
fixed